nexus/src/app/sagas/instance_delete.rs

```diff
@@ -0,0 +1,106 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
```

Comment:
This is totally an aside... are we supposed to have license headers on every file?

Reply:
I believe so? Seems to be the case for nearly every rust file in Omicron